hermes/file_gen_test.go

```diff
@@ -47,13 +28,32 @@ func TestChecksum(t *testing.T) {
 	}
 }
 
-// Test function for the generateFileName function in file_gen.go
+type FileNameTestUnit struct {
+	id       int
+	checksum string
+	want     string
+}
+
+var test_unit_table [5]FileNameTestUnit
+
 func TestFileName(t *testing.T) {
-	completeTestUnitTable()
+	test_unit_table[0].id = 1
+	test_unit_table[0].checksum = "abba"
```

Comment:
could we replace "abba" with "68f3caf439065824dcf75651c202e9f7c28ebf07" , this should make the checksums look more representative.

Reply:
will do!